core/background/services/scrobbleService.js

```diff
@@ -76,28 +90,47 @@ define([], function () {
 			}
 		},
 
+		/**
+		 * Send now playing notification to each bound scrobbler.
+		 *
+		 * @param song
+		 * @param cb
+		 */
 		sendNowPlaying: function (song, cb) {
 			console.log('ScrobbleService: sendNowPlaying() ' + boundScrobblers.length);
-			boundScrobblers.forEach(function (scrobbler) {
+			boundScrobblers.map(scrobbler => {
 				scrobbler.sendNowPlaying(song, cb);
 			});
 		},
 
+		/**
+		 * Scrobble to each bound scrobbler.
+		 *
+		 * @param song
+		 * @param cb
+		 */
 		scrobble: function (song, cb) {
 			console.log('ScrobbleService: scrobble() ' + boundScrobblers.length);
-			boundScrobblers.forEach(function (scrobbler) {
+			boundScrobblers.map(scrobbler => {
```

Comment:
Although it works, the `map` function is usually used to apply given function to array items. `forEach` or for..of should be fine for this case.